minecraft_servers/apfelnetzwerk/manifest.json

```diff
@@ -0,0 +1,81 @@
+{
+  "server_name": "apfelnetzwerk",
+  "nice_name": "Apfelnetzwerk",
+  "direct_ip": "play.apfelnetzwerk.de",
+  "server_wildcards": [
+    "%.apfelnetzwerk.de"
+  ],
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "web": "https://apfelnetzwerk.de/",
+    "web_shop": "https://shop.apfelnetzwerk.de/",
+    "web_support": "https://support.apfelnetzwerk.de/",
+    "twitter": "Apfelnetzwerk",
+    "instagram": "Apfelnetzwerk",
+    "discord": "https://discord.gg/apfelnetzwerk"
+  },
+  "gamemodes": {
+    "BedWars": {
```

Comment:
change all the keys to lower-case (the name can stay like this) - for every gamemode